Gemfile

```diff
@@ -187,6 +191,12 @@ group :development, :test do
 
   # Records HTTP requests
   gem 'vcr'
+
+  # Lint ruby files
+  gem 'rubocop', '~> 0.76.0', require: false
+
+  # Lint RSpec files
+  gem 'rubocop-rspec'
```

Comment:
I'm considering removing `rubocop-rspec`. It seems more annoying than it is useful. I'm guessing that's why I didn't see it in tutor backed 😄 .